docs/rest_api/point_groups.rst

```diff
@@ -0,0 +1,338 @@
+     - Point Group description
+   * - ``isPublished``
+     - ``boolean``
+     - Whether the Point Group is published
+   * - ``dateAdded``
+     - ``datetime``
+     - Date/time Point Group was created
+   * - ``createdBy``
+     - ``int``
+     - ID of the user that created the Point Group
+   * - ``createdByUser``
+     - ``string``
+     - Name of the user that created the Point Group
+   * - ``dateModified``
+     - ``datetime/null``
+     - Date/time Point Group was last modified
+   * - ``modifiedBy``
+     - ``int``
+     - ID of the user that last modified the Point Group
+   * - ``modifiedByUser``
+     - ``string``
+     - Name of the user that last modified the Point Group
+
+.. vale off
+
+List Contact Point Groups
+*************************
+
+.. vale on
+
+.. code-block:: php
+
+   <?php
+
+   //...
+   $pointGroups = $pointGroupApi->getList($searchFilter, $start, $limit, $orderBy, $orderByDir, $publishedOnly, $minimal);
+
+.. code-block:: json
+
+    {
+      "total": 4,
+      "pointGroups": [
+        {
+            "id": 47,
+            "name": "Group A",
+            "description": "This is my first Point Group created via API.",
+            "isPublished": true,
+            "dateAdded": "2024-02-29T12:17:52+00:00",
+            "dateModified": null,
+            "createdBy": 2,
+            "createdByUser": "Admin User",
+            "modifiedBy": null,
+            "modifiedByUser": null
+        },
+        ...
+      ]
+    }
+
+.. vale off
+
+**HTTP Request**
+
+.. vale on
+
+
+``GET /points/groups``
+
+**Response**
+
+``Expected Response Code: 200``
+
+See JSON code example.
+
+**Point Group Properties**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Description
+   * - ``total``
+     - ``int``
+     - Count of all Point Groups
+   * - ``id``
+     - ``int``
+     - ID of the Point Group
+   * - ``name``
+     - ``string``
+     - Point Group name
+   * - ``description``
+     - ``string``
+     - Point Group description
+   * - ``isPublished``
+     - ``boolean``
+     - Whether the Point Group is published
+   * - ``dateAdded``
+     - ``datetime``
+     - Date/time Point Group was created
+   * - ``createdBy``
+     - ``int``
+     - ID of the user that created the Point Group
+   * - ``createdByUser``
+     - ``string``
+     - Name of the user that created the Point Group
+   * - ``dateModified``
+     - ``datetime/null``
+     - Date/time Point Group was last modified
+   * - ``modifiedBy``
+     - ``int``
+     - ID of the user that last modified the Point Group
+   * - ``modifiedByUser``
+     - ``string``
+     - Name of the user that last modified the Point Group
+
+
+.. vale off
+
+Create Point Group
+******************
+
+.. vale on
+
+.. code-block:: php
+
+   <?php
+
+   $data = [
+       'name'        => 'Group A',
+       'description' => 'This is my first Point Group created via API.'
+   ];
+
+   $pointGroup = $pointGroupApi->create($data);
+
+Create a new Point Group.
```

Comment:
Is this meant to be a header, or maybe it's extra text that isn't needed? It wasn't clear to me.

Reply:
It doesn't add any value, let's remove it